rawsocketserver.go

```diff
@@ -0,0 +1,70 @@
+package nexus
+
+import (
+	"io"
+	"net"
+	"time"
+
+	"github.com/gammazero/nexus/stdlog"
+	"github.com/gammazero/nexus/transport"
+	"github.com/gammazero/nexus/transport/serialize"
+)
+
+// RawSocketServer handles socket connections.
+type RawSocketServer struct {
+	router Router
+
+	serializer serialize.Serializer
```

Comment:
still doesnt get used

Reply:
LOL - Done!  Of course, server handles whatever supported encoding client asks for.